webpack.config.js

```diff
@@ -32,10 +33,29 @@ const config = {
   plugins: [
     new Dotenv(),
   ],
+};
+
+const client = {
+  entry: `${APP_DIR}/client.js`,
   output: {
     path: BUILD_DIR,
-    filename: 'bundle.js',
+    filename: 'Overview-client.js',
```

Comment:
you should look into extractTextPlugin

Reply:
Cool! I will. But I'm gonna push this branch first then do that in another branch.